cmd/skopeo/layers.go

```diff
@@ -15,10 +17,14 @@ import (
 var layersCmd = cli.Command{
 	Name:      "layers",
 	Usage:     "Get layers of IMAGE-NAME",
-	ArgsUsage: "IMAGE-NAME",
+	ArgsUsage: "IMAGE-NAME [LAYER...]",
 	Action: func(c *cli.Context) error {
+		fmt.Println(`DEPRECATED: skopeo layers is deprecated in favor of skopeo copy `)
```

Comment:
Shouldn’t this go to stderr? I guess I can see a case for either… just something to consider.